lib/node_modules/@stdlib/_tools/changelog/generate/README.md

```diff
@@ -0,0 +1,110 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2024 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# Generate Changelog
+
+> Generate a changelog for a specified package.
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var generate = require( '@stdlib/_tools/changelog/generate' );
+```
+
+#### generate( pkg\[, releaseType] )
+
+Generates a Markdown formatted changelog for a specified package.
+
+```javascript
+var changelog = generate( 'assert/contains' );
```

Comment:
@Planeshifter I believe you omitted the `@stdlib/` prefix based on how publishing works, correct? However, omitting the prefix is not common convention in other tools (e.g., https://github.com/stdlib-js/stdlib/tree/develop/lib/node_modules/%40stdlib/_tools/pkgs/name2standalone).

Additionally, to be clear, this tool seems, based on the docs alone, to be oriented toward `stdlib` packages, correct? The omission of the `@stdlib/` prefix implies this, but visually a dev may not recognize this.

My personal preference would be to require the `@stdlib/` prefix, which should be a minor adjustment in the publish pipeline.

Reply:
In general, I'd argue for writing this tooling as much as possible on the premise that `stdlib` is a potential consumer, but not the only consumer. In general, I'd like to avoid implicit coupling between this tool, for example, and some other tool/script of which a casual reader will know nothing about.